applications/ContactStructuralMechanicsApplication/custom_conditions/paired_condition.h

```diff
@@ -179,21 +184,37 @@ class KRATOS_API(CONTACT_STRUCTURAL_MECHANICS_APPLICATION) PairedCondition
     ///@name Access
     ///@{
 
-    GeometryType::Pointer pGetPairedGeometry()
+    /**
+     * @brief This method returns the parent geometry
+     * @return The slave geometry (slave in the definition of Popp which is the opposite of the standard)
+     */
+    GeometryType& GetParentGeometry()
     {
-        return mpPairedGeometry;
+        return this->GetGeometry();
     }
 
-    const GeometryType::Pointer pGetPairedGeometry() const
+    /**
+     * @brief This method returns the parent geometry (constant version)
+     * @return The slave geometry (slave in the definition of Popp which is the opposite of the standard)
+     */
+    GeometryType const& GetParentGeometry() const
     {
-        return mpPairedGeometry;
+        return this->GetGeometry();
```

Comment:
Shouldn't this return GetGeometry.GetGeometryPart() ?

Reply:
In #5394